src/components/home/Events.jsx

```diff
@@ -1,8 +1,76 @@
+import Container from '@mui/material/Container';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import Paper from '@mui/material/Paper';
+import Box from '@mui/material/Box';
+
+const leftSidePaperStyles = {
+  elevation: 3,
+  sx: {
+    mr: '20%',
+    mb: '40px',
+    borderRadius: '30px'
+  }
+};
+
 const Events = () => {
-  // any logic needed
-  // return (
-  // all the JSX (looks like HTML) to render
-  // )
+  return (
+    <Container maxWidth={false}>
+      <Grid pb={'40px'} container spacing={3} alignItems={'center'}>
+        <Grid item xs={12} md={6}>
+          <Typography pb={'40px'} variant="h4" align="center" gutterBottom>
+            Upcoming Events
+          </Typography>
+          <Paper {...leftSidePaperStyles}>
+            <Box p={2}>
+              <Typography>
+                CODE PDX meets up in-person once a month for a project demo night and onboarding
+                event. We also occasionally do social meetups.
+                <br></br>
+                <br></br>not sure if we need three boxes. I Need to find more content to put here or
+                remove a bubble and maybe toss in a blob since this will be mid page.
+              </Typography>
+            </Box>
+          </Paper>
+          <Paper elevation={3} sx={{ ml: '20%', mb: '40px', borderRadius: '30px' }}>
```

Comment:
I'm not really a fan of this.  I would prefer just making a component that determines `ml` or `mr` somehow, so it's easier to re-use and maintain if want to make changes in future.   But ultimately it may be more lines of code.

First thought is something like:
```js
const upcomingEventsInfo = [
    {
       text: 'blah blah blah 1' 
    },
    {
        text: 'blah blah blah 2'
    },
    {
        text: 'blah blah blah 3'
    }
];

upcomingEventsInfo.map((item, index) => {
    <Paper 
    key=item.text
    elevation=3,
    sx={{
        mr: {index % 2 === 0 ? 20% : null},
        ml: {index % 2 === 0 ? null : 20%},
        mb: '40px',
        borderRadius: '30px'
    }}>
        <Typography>{item.text}</Typography>
    </Paper>
})
```

Reply:
no change for now. seems like over engineering IMO. There's a high probability we will only have two cards, I just thought three made it look better. Unless we auto populate content into the 3rd one I don't know what will go in there. Also no more than these 3 will ever be added to this section. This logic will certainly apply to the sponsors section to make it easier to render those since they will likely change or more may be added.